colossus/src/main/scala/colossus/protocols/http/streaming/StreamingHttpClient.scala

```diff
@@ -18,19 +18,22 @@ object StreamingHttpClient {
   val client = new ClientFactory[StreamingHttp, Callback, StreamingHttpClient, WorkerRef] {
     def defaultName = "streaming-http-client"
 
-    def apply(config: ClientConfig)(implicit worker: WorkerRef): StreamingHttpClient = {
-      val client = new ServiceClient[StreamingHttp](config, worker.generateContext())
-      val handler = new UnbindHandler(
-        new Controller[Encoding.Client[StreamHttp]](
-          new HttpStreamClientController(client),
-          new StreamHttpClientCodec
-        ),
-        client
-      )
-      worker.worker ! WorkerCommand.Bind(handler)
-      new BasicLiftedClient(client, Some(config))(implicitly[AsyncBuilder[Callback, WorkerRef]].build(worker))
-      with StreamingHttpClient
-    }
+    override def apply(config: ClientConfig)(implicit worker: WorkerRef): StreamingHttpClient = {
+      // TODO not sure how this will work under a load balancer
```

Comment:
Presumably we need to send to the same client for streaming.... spent zero time thinking about this.